communities/epfl-hackers/community-assessment.md

```diff
@@ -0,0 +1,87 @@
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+  - Students interested in working on cool tech projects outside of academia at EPFL.
+  
+### Size
+- How many people are currently in your community?
+  - Hard to tell as there is no official club or society, but I would estimate we are around 5 people.
```

Comment:
An entity of some kind, even if it's just a name, really helps spread the word. Having a brand or group to unite around and affiliate ones self with is helpful for people.